diablo/__init__.py

```diff
@@ -32,7 +32,7 @@
 from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy.exc import SQLAlchemyError
 
-__version__ = '1.2.1'
+__version__ = '1.2.2'
```

Comment:
@johncrossman This change wasn't in the master PR - intentional?

Reply:
@lyttam - Yes, this is a second commit on the PR. We're anticipating a patch.